packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -90,6 +97,10 @@ class ListTileTheme extends InheritedWidget {
   /// If specified, the text color used for enabled [ListTile]s that are not selected.
   final Color textColor;
 
+  /// The padding surrounding the [ListTile]'s titles and its leading and trailing
```

Comment:
When I first read this, I was thinking that it specified the same padding for each of the items in the list tile separately, meaning that if you set it to 10.0, then there would be 20 pixels horizontally between the leading widget and the titles.  It looks like this doesn't do that, but rather puts padding around the outside of the entire widget only, not the individual components (so setting it to 10.0 wouldn't affect internal horizontal padding). You might clear up the wording a bit.